snuba/migrations/migrate.py

```diff
@@ -56,28 +55,30 @@ def run() -> None:
 
     # Create the tables for all of the storages to be migrated.
     for storage_key in STORAGES_TO_MIGRATE:
-        storage_name = storage_key.value
-        logger.info("Creating tables for storage %s", storage_name)
-        storage = get_storage(storage_key)
-        conn = storage.get_cluster().get_query_connection(
-            ClickhouseClientSettings.MIGRATE
-        )
+        run_storage(storage_key)
+
+
+def run_storage(storage_key: StorageKey) -> None:
```

Comment:
Adding this method was needed to selectively apply creation statement and migrations to an individual storage.  It is needed in tests/base.py